mrp_analytic/models/analytic_account.py

```diff
@@ -0,0 +1,17 @@
+# © 2015 Pedro M. Baeza - Antiun Ingeniería
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models
+from odoo import fields
+
+
+class AnalyticAccount(models.Model):
+    _inherit = "account.analytic.account"
+
+    def _compute_num_productions(self):
+        for analytic_account in self:
+            analytic_account.num_productions = self.env['mrp.production'].search_count([
```

Comment:
Travis says this line is too long. Find a way to write it within 80 characters, either splitting it on multiple lines or (best way, IMHO) renaming `analytic_account` into something shorter (`an_acc` or `record` should make the job)